alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/accessor/SlackJobDetailsAccessor.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * alert-common
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.persistence.accessor;
+
+import com.synopsys.integration.alert.common.persistence.model.job.details.SlackJobDetailsModel;
+
+public interface SlackJobDetailsAccessor extends JobDetailsAccessor<SlackJobDetailsModel> {
```

Comment:
Do we need this interface? Should we just have the implementation of the SlackJobDetailsAccessor? Or is this interface being used across sub-projects?

Reply:
@gkillough do we? I followed what was done with Email V2, I figured it had something to do with Spring auto-wiring.